### DISCUSSION TITLE
Feature Request: Add config option to use data.page.date

### DISCUSSION THREAD

**Discussion by DougReeder** [web-alchemy/eleventy-plugin-content-dates#4] [Ideas]
Checking git is expensive when there are many files, and the exact dates rarely matter except for the final build that users see. It speeds performance to use `data.page.date` instead.

I propose a boolean configuration option that when enabled would return `data.page.date`, if a date field is `Date. Git. Created` or `Date. Git. Last Modified`

This being a configuration option, the user could use, for example `data.eleventy.env.runMode !== 'build'` or `process.env.NODE_ENV === 'production'` or whatever criterion met their use case.

**Comment by monochromer** [comment]
It is not obvious, but you can use [`11tydata.js` files with computed field](https://github.com/web-alchemy/eleventy-plugin-content-dates?tab=readme-ov-file#low-level-usage-example). Example:

```js
import eleventyPluginContentDates from '@web-alchemy/eleventy-plugin-content-dates';
const {
  TIMESTAMPS,
  computeDate,
  getContentFolderPath,
  getContentFilePath
} = eleventyPluginContentDates;

module.exports = {
  eleventyComputed: {
    updatedAt(data) {
      return computeDate({
        strategy: data.eleventy.env.runMode !== 'build' ? TIMESTAMPS.FS_LAST_MODIFIED : TIMESTAMPS.GIT_LAST_MODIFIED,
        // get date for main template file
        contentPath: getContentFilePath
       // or get date for all folder containing template file
       contentPath: getContentFolderPath
      })
    }
  }
}
```

**Reply by DougReeder** [reply]
Yes, that's what I've done for now, but the config option would be nice for webmasters who aren't programmers.
